src/Compilers/VisualBasic/Portable/Lowering/SyntheticBoundNodeFactory.vb

```diff
@@ -357,6 +357,24 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return Block(locals, ImmutableArray.Create(Of BoundStatement)(statements))
         End Function
 
+        Public Function StatementList(first As BoundStatement, second As BoundStatement) As BoundStatementList
```

Comment:
Two signatures for StatementList survive: this one, with exactly two statements, and one that takes an immutable array of statements.